pkg/model/serviceinstance.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
```

Comment:
I think the right go convention is all lowercase:
like in library: import "github.com/golang/example/stringutil"
like in mixer: servicegraph.go
and add _test for tests. This will look like serverinstance.go and serverinstance_test.go